komodor/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "995fe904-e761-4f2f-8dbf-148baf3f080a",
+  "app_id": "komodor",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Track changes across your entire K8s landscape and stack",
+    "title": "Komodor Automation",
+	  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+	"Category::Troubleshooting",
+    "Offering::Integration"
+  ]
```

Comment:
```suggestion
    "title": "Komodor Automation",
    "classifier_tags": [
      "Supported OS::Linux",
      "Supported OS::Mac OS",
      "Supported OS::Windows",
      "Offering::Integration"
    ]
```
Small nit, can we please fix the indentation here? Also, it appears that `Category::Troubleshooting` is not a valid classifier tag at this time. Can we please remove it for now? It was causing this validation to fail:

```
Error validating manifest schema:
Value of parameter 'classifier_tags' should be any of these ['Category::Alerting', 'Category::Autodiscovery', 'Category::Automation', 'Category::AWS', 'Category::Azure', 'Category::Caching', 'Category::Cloud', 'Category::Collaboration', 'Category::Compliance', 'Category::Configuration & Deployment', 'Category::Containers', 'Category::Cost Management', 'Category::Data Store', 'Category::Developer Tools', 'Category::Event Management', 'Category::Exceptions', 'Category::Google Cloud', 'Category::Incidents', 'Category::IOT', 'Category::ISP', 'Category::Issue Tracking', 'Category::Languages', 'Category::Log Collection', 'Category::Mainframe', 'Category::Marketplace', 'Category::Messaging', 'Category::Metrics', 'Category::Mobile', 'Category::Monitoring', 'Category::Network', 'Category::Notification', 'Category::Oracle', 'Category::Orchestration', 'Category::OS & System', 'Category::Processing', 'Category::Profiling', 'Category::Provisioning', 'Category::SAP', 'Category::Security', 'Category::SNMP', 'Category::Source Control', 'Category::Testing', 'Category::Tracing', 'Category::Web', 'Offering::Integration', 'Offering::Professional Service', 'Offering::Software License', 'Offering::UI Extension', 'Supported OS::Android', 'Supported OS::Any', 'Supported OS::HP-UX', 'Supported OS::IBM i/OS', 'Supported OS::IBM z/OS', 'Supported OS::iOS', 'Supported OS::Linux', 'Supported OS::macOS', 'Supported OS::Solaris', 'Supported OS::Windows', 'Supported OS::Mac OS', 'Category::OS System']
```